app/javascript/mastodon/features/account/components/header.js

```diff
@@ -16,6 +16,7 @@ const messages = defineMessages({
   unblock: { id: 'account.unblock', defaultMessage: 'Unblock @{name}' },
   edit_profile: { id: 'account.edit_profile', defaultMessage: 'Edit profile' },
   linkVerifiedOn: { id: 'account.link_verified_on', defaultMessage: 'Ownership of this link was checked on {date}' },
+  account_locked: { id: 'account.locked_info', defaultMessage: 'This account privacy status is set to locked. The owner manually reviews who can follow them.' },
```

Comment:
Hey over here, shouldn't be `[...] who can follow him.` instead of `[...] who can follow them.` ?

Bye :wave:

Reply:
Yes, it's meant to be gender-neutral

Reply:
Sorry for the oversight. Should I make a new PR fixing it?

Reply:
Oh, I just realized I read this the other way, I used gender neutral `them`. Please disregard my comment.